pywheel/shim/_binwrapper.py

```diff
@@ -37,6 +41,9 @@ def _config_exe(exe_name):
     # add nmodl home to environment (i.e. necessary for nrnunits.lib)
     os.environ["NMODLHOME"] = NMODL_HOME
 
+    # set PYTHONPATH for embedded python to properly find the nmodl module
+    os.environ["PYTHONPATH"] = working_set.by_key[package_name].location + ':' + os.environ.get("PYTHONPATH", "")
```

Comment:
yes but... why do we need this now?

Reply:
I think we may have just not tested this properly in the usecase of wheel+python venv. Why it works on linux and not on mac without the PYTHONPATH set is unclear to me too. But I believe the patch does fix the issue and it makes sense given that embedded python instances are known to have trouble finding site-packages modules (https://bugs.python.org/issue22213)

Reply:
As discussed on slack, embedded python has some weird (platform specific) behaviour.